### ISSUE TITLE
selfhost invidious on docker, yatte refuse display thumbnail

### ISSUE THREAD

**Issue by Yumega** [yattee/yattee#520]
### Guidelines

- [ ] I have searched the [issue tracker](https://github.com/yattee/yattee/issues) and I haven't found bug report like this

### Current Behavior

if I visit my selfhost invidous http://172.19.0.3:3000/ through browser like chrome firefox, there is no problem
but if I use it on yattee ,there is no thumbnails, but I can play all the videos. 

### Expected Behavior

display thumnail

### Device & OS Version

iphone and appleTV

### App Build

1.4.5

**Comment by swolford** [comment]
> 

I was having the same issue, except I have SSL through linuxserver SWAG. I applied this to my docker-compose, and changed to IP instead of domain in the app and it worked. +1 for bug.

**Comment by cwallace** [comment]
Thanks for the work around, but I'd prefer not to expose additional ports just to grab thumbnails. Like @swolford I have invidious reverse proxied using NPM.

**Comment by steilerDev** [comment]
As someone who ran into this - I don't think this is an issue with yattee (maybe a documentation issue at best)

From the [invidious docs](https://github.com/iv-org/invidious/blob/master/config/config.example.yml):

```
##
## When the invidious instance is behind a proxy, and the proxy
## listens on a different port than the instance does, this lets
## invidious know about it. This is used to craft absolute URLs
## to the instance (e.g in the API).
##
## Note: This setting is MANDATORY if invidious is behind a
## reverse proxy.
##
## Accepted values: 1-65535
## Default: <none>
##
#external_port:
```

```
##
## Domain name under which this instance is hosted. This is
## used to craft absolute URLs to the instance (e.g in the API).
## The domain MUST be defined if your instance is accessed from
## a domain name (like 'example.com').
##
## Accepted values: a fully qualified domain name (FQDN)
## Default: <none>
##
domain:
```

So this is a requirement of the invidious API when running the instance behind a proxy - this is not exposing additional ports, simply instructing the API to use the correct ones.

PSA: if you are running invidious in a separate container network and use something like [`nginx-proxy`](https://hub.docker.com/r/nginxproxy/nginx-proxy) to expose it to your network under a subdomain, your `docker-compose` should look something like this:
```
invidious:
    image: quay.io/invidious/invidious:latest
    restart: unless-stopped
    container_name: invidious
    environment:
      VIRTUAL_HOST: invidious.steilerdev.de
      VIRTUAL_PORT: 3000
      LETSENCRYPT_HOST: invidious.steilerdev.de
      # Please read the following file for a comprehensive list of all available
      # configuration options and their associated syntax:
      # https://github.com/iv-org/invidious/blob/master/config/config.example.yml
      INVIDIOUS_CONFIG: |
        db:
          <db-config>
        check_tables: true
        hmac_key: <hmac-key>
        domain: invidious.steilerdev.de
        external_port: 443
        https_only: true
```

**Comment by swolford** [comment]
I reverted my configuration, and the only thing different was `external_port`. I set it to 443 and it works! I was initially confused because the wording on the invidious docs is not very clear so I assumed it was referring to the port that the proxy manager uses for the container, and not the _actual_ SSL port. Since the proxy manager wasn't giving me errors, I had also assumed it was correctly configured.

> As someone who ran into this - I don't think this is an issue with yattee (maybe a documentation issue at best)

I went to see if anything could be done about adding it to the yattee docs, and can see that this has already been answered in the wiki under FAQ/General/Point 6. I should've checked. :)

https://github.com/yattee/yattee/wiki/FAQ#general

I think this can be closed.
